app/src/main/java/net/gsantner/markor/ui/hleditor/Highlighter.java

```diff
@@ -73,6 +74,7 @@ protected static Highlighter getDefaultHighlighter(HighlightingEditor hlEditor,
     protected boolean _highlightLinks = true;
     protected final boolean _highlightHexcolor;
     protected final Document _document;
+    private TextWatcher _modifier = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5359905)